test/ctia/domain/id_test.clj

```diff
@@ -0,0 +1,73 @@
+(ns ctia.domain.id-test
```

Comment:
I secretly despise clojure.test fixtures because of the way they glom onto everything in a ns.